cookbooks/ros2_windows/recipes/pip_installs.rb

```diff
@@ -22,16 +21,21 @@
 ]
 
 ros2doctor_network_dependency = {
-  "foxy" => "ifcfg",
-  "galactic" => "ifcfg",
```

Comment:
Slightly unthrilled by the cleanup being lumped in with the explicit changes but not enough to block it.